src/index.js

```diff
@@ -0,0 +1,104 @@
+// Temperature //
+const state = {
+  tempByDegree: 70,
+};
+
+const clickButtonUp = () => {
+  state.tempByDegree += 1;
+  const tempElement = document.querySelector('#temp');
+  tempElement.textContent = `${state.tempByDegree}`;
+  color = changeTempColor();
+  tempElement.className = color;
+  //landscape
+  const landElement = document.querySelector('#landscape');
+  landscape = lanscapeScale();
+  landElement.textContent = landscape;
+};
+const tempUp = document.querySelector('#tempUp');
+tempUp.addEventListener('click', clickButtonUp);
+
+const clickButtonDown = () => {
+  state.tempByDegree -= 1;
+  const tempChanger = document.querySelector('#temp');
+  tempChanger.textContent = `${state.tempByDegree}`;
+  color = changeTempColor();
+  tempChanger.className = color;
+  //landscape
+  const landElement = document.querySelector('#landscape');
+  landscape = lanscapeScale();
+  landElement.textContent = landscape;
+};
+
+const tempDown = document.querySelector('#tempDown');
+tempDown.addEventListener('click', clickButtonDown);
+
+//change temp range color
+const changeTempColor = () => {
+  let color = '';
+  if (state.tempByDegree < 49) {
+    color = 'teal';
```

Comment:
As I mentioned in office hours, it's a good idea to avoid class names like this going forward.  Not a big deal, just a good idea to future proof things.  :smiley: